go/chat/server.go

```diff
@@ -1786,14 +1789,19 @@ func (h *Server) Sign(payload []byte) ([]byte, error) {
 }
 
 func (h *Server) postAttachmentPlaceholder(ctx context.Context, arg postAttachmentArg) (chat1.PostLocalRes, error) {
-	// generate outbox id
-	rbs, err := libkb.RandBytes(8)
-	if err != nil {
-		return chat1.PostLocalRes{}, err
+	if arg.OutboxID == nil {
+		// generate outbox id
+		rbs, err := libkb.RandBytes(8)
```

Comment:
Can you call `storage.NewOutboxID()` here?

Reply:
I realize I had it like this before, but I think that was a mistake.

Reply:
done